jmx2ini/jmx2ini.go

```diff
@@ -0,0 +1,306 @@
+package main
+
+import (
+	"fmt"
+	"launchpad.net/xmlpath"
```

Comment:
Is "launchpad.net/xmlpath" the only way to do this?
Seems like a pesky dependency since it requires Bazaar to 'go get'  it.
go: missing Bazaar command. See https://golang.org/s/gogetcmd